daniel-v2/README.md

```diff
@@ -27,10 +28,18 @@
   - ✅✨ [Sitemap](https://wpdeveloperstaging.wordpress.com/docs/platform-features/sitemaps/)
   - ✅✨ [Jetpack Scan](https://wpdeveloperstaging.wordpress.com/docs/platform-features/jetpack-scan/)
   - ✅✨ [Account Security](https://wpdeveloperstaging.wordpress.com/docs/platform-features/account-security/)
+  - 🚧 [Automatic Data Center Failover](https://github.com/Automattic/dotcom-content/issues/218)
+  - 🚧 [Distributed Denial of Service (DDOS) and Web Application Firewall (WAF) protection](https://github.com/Automattic/dotcom-content/issues/219)
+  - 🚧 [Jetpack Search](https://github.com/Automattic/dotcom-content/issues/220)
+  - 🚧 [Akismet Anti-Spam](https://github.com/Automattic/dotcom-content/issues/221)
+  - 🚧 [Jetpack SEO Tools](https://github.com/Automattic/dotcom-content/issues/222)
+  - 🚧 [Cloudflare Integration](https://github.com/Automattic/dotcom-content/issues/223)
+  - 🚧 [Jetpack High-Level Walkthrough](https://github.com/Automattic/dotcom-content/issues/225)
```

Comment:
Have we considered adding some of those features in the sitemap? For example, `Brute force protection`, `Downtime monitoring`, `Boost`.  Whenever I deploy a new WP site, I use plugins or other products for these things. It would be great letting people know we got them covered right out of the box.

Reply:
That could be a good alternative; it highlights those features much more. What do you think about extracting the following?

- [Brute force protection](https://jetpack.com/support/protect/)
- [Downtime monitoring](https://jetpack.com/support/monitor/)
- [Boost](https://jetpack.com/support/jetpack-boost/)
- [WP.com login](https://jetpack.com/support/sso/)

Reply:
I added https://github.com/Automattic/dotcom-content/issues/225 with the same motivation, I think it would be great to highlight Jetpack features, and to avoid developers disabling Jetpack and installing a bunch of alternative plugins for the same features.

Jetpack does so many things that I didn't want to create that many separate issues, but it'd be great to highlight these four as separate sections.

Reply:
I asked @robertbpugh to weigh in here!

Reply:
I agree. The original idea was to encompass all functionalities in one document, but after some feedback, we have split some features into separate documents, so it might be better to give each feature its own document.